### DISCUSSION TITLE
Entra Id and V5 Corporate Proxy Patches

### DISCUSSION THREAD

**Discussion by SolidAnonDev** [nextauthjs/next-auth#10993] [Help]
Since we still can't have inbuilt support for a proxy it seems, does anyone know where/how to modify the library source with HttpProxyAgent to allow for this?

[In V4 I was doing what is outlined in this web page, and this worked fine. ](https://next-auth.js.org/tutorials/corporate-proxy)

However, upon upgrading to V5, it is not clear to me where an HttpProxyAgent should be instantiated for Entra Id (or any provider for that matter)

Upon deploying, my fetch requests are failing, very similar to how they would in V4. I can find the profile photo call, and I actually just remove that because I don't need it, but I don't know where the open id client or something similar would be.

**Comment by ChristianRaoulis** [comment]
I got the same issue. Has someone a working package patch?
